### PR TITLE
Deployments deprecation notice.

### PR DESCRIPTION
**Pull request by jubeira**
# Description

As an intermediate step before removing the workspace completely, this PR adds a deprecation notice while the announcements roll out.
See #2504.

## Type of change

- [ ] Bug fix 
- [ ] New feature 
- [ ] Breaking change 
- [ ] Dependency changes
- [ ] Code refactor / cleanup
- [x] Documentation or wording changes
- [ ] Other

## Checklist:

- [x] The diff is legible and has no extraneous changes
- N/A Complex code has been commented, including external interfaces
- N/A Tests are included for all code paths
- [x] The base branch is either `master`, or there's a description of how to merge

## Issue Resolution

N/A

### CODE REVIEW

File: pkg/deployments/README.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 # Balancer V2 Deployments
 
+> ⚠️ This workspace has been moved to https://github.com/balancer/balancer-deployments, and is no longer maintained here.
+>
+> Please start using `balancer-deployments` repository as a source of truth for existing and upcoming deployments.
+> This workspace will be removed from the monorepo by 2023-06-09.
+
 [![NPM Package](https://img.shields.io/npm/v/@balancer-labs/v2-deployments.svg)](https://www.npmjs.org/package/@balancer-labs/v2-deployments)
```
Comment (EndymionJkb) [inline pkg/deployments/README.md:10]: Do we have NPMs from the new repo yet? If not, I'd say remove this for now. (Unless the previous NPM is the only source?)

**Review by EndymionJkb — COMMENTED** [review]
We also need to update the main readme, since that also has links and comments about 'v2-deployments'

File: pkg/deployments/README.md (hunk 1 above)
Comment (jubeira) [reply pkg/deployments/README.md:10]: The source should change anything. NPM just packs things and publishes them on demand if I'm not mistaken.

File: pkg/deployments/README.md [hunk 2]
```diff
@@ -2,6 +2,11 @@
...
 [![GitHub Repository](https://img.shields.io/badge/github-deployments-lightgrey?logo=github)](https://github.com/balancer-labs/balancer-v2-monorepo/tree/master/pkg/deployments)
```
[the remark this answers is not in the document]
Comment (jubeira) [reply pkg/deployments/README.md:11]: Hmm that's the link in the badge, I'm not sure whether we should start pointing to the new one while keeping stuff here.

I'm inclined to leave everything as-is with the deprecation notice. We are already removing everything and updating the references in #2504.

To me this is like when we deprecate a task: we keep it as is, and we just add a warning at the top. WDYT?

[the remark this answers is not in the document]
Comment (EndymionJkb) [reply pkg/deployments/README.md:11]: Ok; if it's change in the other PR it's fine here then, I guess. Though the deployments repo is already more recent than this one. Whatever you think is best.

**Review by EndymionJkb — APPROVED** [review]

File: pkg/deployments/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (EndymionJkb) [reply pkg/deployments/README.md:11]: The other PR also has changes in the main readme?

[the remark this answers is not in the document]
Comment (jubeira) [reply pkg/deployments/README.md:11]: The other PR just cleans up everything, and leaves a note that says where the new home is :)

**Pull request merged**